src/Mod/Draft/draftobjects/shape2dview.py

```diff
@@ -149,21 +149,19 @@ def getProjected(self,obj,shape,direction):
         edges = []
         _groups = TechDraw.projectEx(shape, direction)
         for g in _groups[0:5]:
-            if g:
+            if not g.isNull():
```

Comment:
I think g could be None
```suggestion
            if g and not g.isNull():
```